src/main/rx/rx.h

```diff
@@ -114,6 +114,7 @@ typedef struct rxConfig_s {
     uint8_t spektrum_sat_bind;              // number of bind pulses for Spektrum satellite receivers
     uint8_t rssi_channel;
     uint8_t rssi_scale;
+    uint8_t rssi_warningLevel;
```

Comment:
yes correct.
but you're right, a separate PR is preferable.

Le 05/01/2016 12:37, Petr Ledvina a écrit :
>
> In src/main/rx/rx.h 
> <https://github.com/cleanflight/cleanflight/pull/1654#discussion_r48836391>:
>
> > @@ -114,6 +114,7 @@ typedef struct rxConfig_s {
> >      uint8_t spektrum_sat_bind;              // number of bind pulses for Spektrum satellite receivers
> >      uint8_t rssi_channel;
> >      uint8_t rssi_scale;
> > +    uint8_t rssi_warningLevel;
>
> is this correct? This will break configs in eeprom, at least version 
> must be bumped. Also separate PR is reasonable ..
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cleanflight/cleanflight/pull/1654/files#r48836391>.
>